content/en/docs/tutorials/security/apparmor.md

```diff
@@ -393,15 +413,15 @@ Specifying the profile a container will run with:
 
 ### Profile Reference
 
-- `runtime/default`: Refers to the default runtime profile.
+- `RuntimeDefault`: Refers to the default runtime profile.
   - Equivalent to not specifying a profile, except it still
     requires AppArmor to be enabled.
   - In practice, many container runtimes use the same OCI default profile, defined here:
     https://github.com/containers/common/blob/main/pkg/apparmor/apparmor_linux_template.go
-- `localhost/<profile_name>`: Refers to a profile loaded on the node (localhost) by name.
+- `Localhost`: Refers to a profile loaded on the node (localhost) by name.
   - The possible profile names are detailed in the
     [core policy reference](https://gitlab.com/apparmor/apparmor/wikis/AppArmor_Core_Policy_Reference#profile-names-and-attachment-specifications).
-- `unconfined`: This effectively disables AppArmor on the container.
+- `Unconfined`: This effectively disables AppArmor on the container.
```

Comment:
Does this belong in the tutorial page? I'd expect it to live inside https://kubernetes.io/docs/reference/node/

Also, this could be a description list - see https://discourse.gohugo.io/t/description-list-shortcode/9787/4 for an example.

Reply:
Changed to description list in [91e1830](https://github.com/kubernetes/website/pull/37191/commits/91e1830d8d414285e5d5e05b94de3f5f4f4fc94f)